src/canmatrix/tests/test_sym.py

```diff
@@ -46,3 +46,27 @@ def test_colliding_mux_values():
 
     assert 'FirstMux' in error_string
     assert 'SecondMux' in error_string
+
+
+def test_parse_longname():
```

Comment:
```suggestion
def test_parse_longname_with_colon():
```
Delineates it from a 'generic' long name parsing test.